buildah.go

```diff
@@ -136,6 +137,34 @@ func GetBuildInfo(b *Builder) BuilderInfo {
 	}
 }
 
+// CommonBuildOptions are reseources that can be defined by flags for both buildah from and bud
```

Comment:
FWIW, a nice approach, I like it.  Much better than dumping them all into BuilderOptions individually.